### PR TITLE
Email validation of the Form

### PR DESCRIPTION
**Pull request by RHK-MICROVERSE**
Java script has been added for email validation of the Form.

### CODE REVIEW

**Review by didierganthier — COMMENTED** [review]
Hi Team,

While you put in a lot of hard work in this project, unfortunately, I cannot proceed with the review of your code.



## Invalid Code Review Request 🚫

- This milestone is a [pair-programming activity](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/6_accessability.md#work-and-submission-mode) yet the commit history shows only one contributor.

Your Code Review Request will be marked as invalid in your Dashboard, but don't worry you can always try again! 💪

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

--------

_Invalid Code Review Request **does not count** towards the code reviews limit._

**Review by wayungi — COMMENTED** [review]
Hi @RHK-MICROVERSE , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project which has missing parts of the README. Kindly use this [readme template](https://github.com/microverseinc/readme-template)  and fill in all sections

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by AlphaNtihinduka — COMMENTED** [review]
Hello Team,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request ❌ 
usb-invalid

- [ ] You have submitted a project with a readme file that is missing the `key features`, `future features` and `the installation and running` of your application. If some of the parts are not available, you may consider stating it. Example : `Installation is not necessary`. Kindly edit and customize your readme with the project information for it to be professional.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray

Cheers and Happy coding!clapclapclap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Comment by addod19** [comment]
### Hi **TEAM**, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is invalid because we have eslint linter errors which must be fixed before submitting a new code review request

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by codecaiine — COMMENTED** [review]
Hi @RHK-MICROVERSE , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
You have submitted a project, that has not completed the readme file.

- All main headers from the README template should be present. Even if there's no information for a section or sub-section, rather leave texts like "NA", "None" or "coming soon" than delete it. 
- Future Features section should include 1-3 potential features to be added. 
- Each section description should be removed and replaced with their custom content. The section descriptions are meant to guide you in documenting your README. They are not meant to be part of the final documentation. 
- The License suggestion should be removed. 
- There should not be any blank sections that do not have content. 
- There should not be any broken links in the README file. 
- Delete the live demo section if the application is not deployed. If project requirements say to deploy the project, the section should be restored and the link updated. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: js/script.js [hunk 1]
```diff
@@ -210,4 +210,26 @@ function controlMobileMenu() {
...
+  if ((emailInput.value).match(/[A-Z]/)) {
+    emailErrorMessage.style.display = 'block';
+    submitButton.type = 'button';
+    e.preventDefault();
+  } else {
+    emailErrorMessage.style.display = 'none';
+    submitButton.type = 'submit';
+  }
+}
+submitButton.addEventListener('click', validationError);
```
Comment (tresorsawasawa) [inline js/script.js:235]: - [ **OPTIONAL** ] Good :+1: job with your email validation as it works as expected :100:. However, I noticed that the error message shows up even if before filling any input field(i.e. just when a user launches your app.). I suggest you update your code in the way the error message only shows up when the user put `email` with uppercase characters.

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @RHK-MICROVERSE and @kalbek 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**